package.json

```diff
@@ -7,33 +7,33 @@
     "test": "gulp test"
   },
   "dependencies": {
-    "font-awesome": "4.3.0",
     "bootstrap": "3.3.2",
     "bootstrap-material-design": "fezvrasta/bootstrap-material-design#0.3.0",
-    "underscore": "1.8.2",
     "camelcase": "1.0.2",
     "classnames": "2.1.3",
     "es6-promise": "2.3",
     "flux-react": "2.6.4",
     "font-awesome": "4.3.0",
     "htmlparser2": "3.8.3",
     "jquery": "2.1.4",
+    "katex": "0.5.1",
     "markdown-it": "4.1.1",
     "mime-types": "2.1.3",
     "moment": "2.9.0",
     "moment-timezone": "0.4.0",
-    "openstax-react-components": "openstax/react-components#53ed6ed38b6752523e85104c3f7ef7c5c36debc8",
+    "openstax-react-components": "openstax/react-components#bc7d2a1bd2a48cd991920f250dad8a3aadcd8ce0",
     "pluralize": "1.1.2",
+    "quill-with-math": "philschatz/quill-with-math#master",
```

Comment:
I guess so?? Those aren't new, the changes are the result of the sorting the dependencies, when I installed `react-widgets` (for it's multi-select).